<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsDialog</class>
 <widget class="QDialog" name="SettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>350</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <property name="windowIcon">
   <iconset theme="preferences-system">
    <normaloff>.</normaloff>.</iconset>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <property name="fieldGrowthPolicy">
    <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
   </property>
   <property name="labelAlignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
   <property name="formAlignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
   <item row="0" column="0">
    <widget class="QLabel" name="sortLabel">
     <property name="text">
      <string>Sort files by</string>
     </property>
     <property name="buddy">
      <cstring>sortCombo</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QComboBox" name="sortCombo">
     <item>
      <property name="text">
       <string>Name alphabetically</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Modification date</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Natural sorting</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="nlineLabel">
     <property name="text">
      <string>New line character</string>
     </property>
     <property name="buddy">
      <cstring>nlineCombo</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QComboBox" name="nlineCombo">
     <item>
      <property name="text">
       <string notr="true">LF (Unix)</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">CR/LF (Windows)</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">CR (Classic Mac)</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="encodingLabel">
     <property name="text">
      <string>Text encoding</string>
     </property>
     <property name="buddy">
      <cstring>encodingCombo</cstring>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QComboBox" name="encodingCombo">
     <property name="editable">
      <bool>true</bool>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <property name="placeholderText">
      <string>Select encoding name</string>
     </property>
     <item>
      <property name="text">
       <string notr="true">UTF-8</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">Windows-1250</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="dpiLabel">
     <property name="text">
      <string>Required DPI</string>
     </property>
     <property name="buddy">
      <cstring>dpiSpin</cstring>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QSpinBox" name="dpiSpin">
     <property name="suffix">
      <string notr="true"> DPI</string>
     </property>
     <property name="minimum">
      <number>100</number>
     </property>
     <property name="maximum">
      <number>2400</number>
     </property>
     <property name="value">
      <number>300</number>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="checksumLabel">
     <property name="text">
      <string>Chechsum type</string>
     </property>
     <property name="buddy">
      <cstring>checksumCombo</cstring>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QComboBox" name="checksumCombo">
     <property name="currentIndex">
      <number>2</number>
     </property>
     <item>
      <property name="text">
       <string notr="true">MD4</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">MD5</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">SHA-1</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">SHA-224</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">SHA-256</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">SHA-348</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">SHA-512</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">SHA3-224</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">SHA3-256</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">SHA3-384</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">SHA3-512</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">Keccak-22</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">Keccak-256</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">Keccak-384</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">Keccak-512</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="formatLabel">
     <property name="text">
      <string>Chechsum format</string>
     </property>
     <property name="buddy">
      <cstring>formatCombo</cstring>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QComboBox" name="formatCombo">
     <item>
      <property name="text">
       <string notr="true">Base64</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Hexadecimal</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="textLabel">
     <property name="text">
      <string>Additional text</string>
     </property>
     <property name="buddy">
      <cstring>textEdit</cstring>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QLineEdit" name="textEdit">
     <property name="placeholderText">
      <string>Optional file name sufix</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0" colspan="2">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="8" column="0" colspan="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Discard|QDialogButtonBox::Reset|QDialogButtonBox::Save</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>sortCombo</tabstop>
  <tabstop>nlineCombo</tabstop>
  <tabstop>encodingCombo</tabstop>
  <tabstop>dpiSpin</tabstop>
  <tabstop>checksumCombo</tabstop>
  <tabstop>formatCombo</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
